cypress/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+const config = require('../.eslintrc.js')
+
+module.exports = {
+    ...config,
```

Comment:
I don't think you have to merge the configs manually here. Eslint will automatically traverse upwards and merge the configs it finds: https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy. It'll traverse upwards until it encounters an eslintrc with `"root": true`

Reply:
Might be that this pattern was borrowed from the `cli-style` configs, but they are a special case.

Reply:
I see that this was resolved, but the manual merge is still in there.

Reply:
I was looking at an older file. Was fixed and resolved correctly.